nautobot/dcim/models/device_components.py

```diff
@@ -433,6 +433,7 @@ class PowerOutlet(CableTermination, PathEndpoint, ComponentModel):
         null=True,
         related_name="poweroutlets",
     )
+    # todoindex:
```

Comment:
```
dcim/models/device_components.py-377-                # Setup numerator and denominator for later display.
dcim/models/device_components.py-378-                for leg, leg_name in PowerOutletFeedLegChoices:
dcim/models/device_components.py:379:                    outlet_ids = PowerOutlet.objects.filter(power_port=self, feed_leg=leg).values_list("pk", flat=True)
dcim/models/device_components.py:380:                    utilization = PowerPort.objects.filter(
dcim/models/device_components.py-381-                        _cable_peer_type=poweroutlet_ct, _cable_peer_id__in=outlet_ids
dcim/models/device_components.py-382-                    ).aggregate(
```

Reply:
Good candidate since it has a list of choices which increases the likelihood of index hits.